### PR TITLE
Add periodic data

### PR DESCRIPTION
**Pull request by sunzhuohang**
What problem does this PR solve?
Add a function to keep the load running, which is a periodic normal distribution in the time dimension。On the purpose of generating a regular load for testing。

What is changed and how it works?
Add new parameters in prop.go.

period_in_time :
Control this function, the default value is false,which means to ignore this function. 
timedistribution:
Distribution type in time.
expectedvalue :
Mean of normal distribution.
standarddeviation :
Standard deviation of normal distribution.
timedelay :
Qps decreases with increasing latency.
timeperiod :
One cycle duration.
noiseratio:
The noise amplitude, for example, if it's 0.5, which means that the load may change within the range of 0.5-1.5 times itself at most.

Check List
Tests

Manual test
Use this workload file - workload_normal.
QPS and TIKV CPU USAGE are as follows：
image
image

### CODE REVIEW

File: pkg/client/delay.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+import (
+	"math"
+	"math/rand"
+	"time"
+)
+
+var oldtime int = -1
+var noiseRange float64 = 0
+//if radio is y，noiserange is randfloat64 in [-y/(1+y),y/(1-y)]
```
Comment (nolouch) [inline pkg/client/delay.go:11]:
```suggestion
// If radio is y，noiseRange is rand float64 in [-y/(1+y),y/(1-y)]
```

File: pkg/client/delay.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+func getnoiseRange(newtime int,ratio float64){
+	//not locked, not necessary
```
Comment (nolouch) [inline pkg/client/delay.go:13]:
```suggestion
	// Not locked, not necessary
```

**Comment by sunzhuohang** [comment]
Because of different distribution calculation, the parameter timedelay is changed to expected_ops. This parameter set the maximum ops in one cycle duration, but in reality it will not be greater than this value, because there is also a delay in executing operations.

File: pkg/client/delay.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+func (w *worker) Noise_step(loadStartTime time.Time)  {
+	w.delay = int64(float64(w.threadCount) * float64(time.Second) / float64(w.expectedOps))
+	timeNow := int(time.Now().Sub(loadStartTime).Minutes())%w.period
+	v := 3 - int(float64(timeNow * 3.0 /w.period))
+	tmp := float64(v)*float64(w.delay)
+	getnoiseRange(timeNow,w.noiseRatio)
+	noise := tmp*noiseRange
+	time.Sleep(time.Duration(tmp+noise))
+}
```
Comment (lhy1024) [inline pkg/client/delay.go:71]: file end

File: workloads/workload_normal [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+# TimeDistribution = "timedistribution"
+# TimeDistributionDefault = "none"
+
+period_in_time = true
+meanvalue = 30
+standarddeviation = 5
+expected_ops = 40000
+timeperiod = 60
+noiseratio = 0.5
+timedistribution = noise_normal
```
Comment (lhy1024) [inline workloads/workload_normal:121]: file end

**Comment by disksing** [comment]
Do we still need to push it forward? /cc @nolouch @lhy1024

**Pull request closed without merging**